yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -37,6 +37,7 @@ import (
 	_ "github.com/jackc/pgx/v5/stdlib"
 	"github.com/samber/lo"
 	log "github.com/sirupsen/logrus"
+	"github.com/tiendc/go-deepcopy"
```

Comment:
Just as a general discussion, not suggesting you change it:
I'm guessing you picked this because it seems to be the fastest. In our case, though, we don't really care so much about speed (because we call it like once and log it). In such cases, it makes more sense to prioritize dependencies that are well maintained.

Reply:
Not just fast, I also looked at activity on the repo and only this one seems quite active, rest had last commit years ago